data/pref.xml

```diff
@@ -425,6 +425,7 @@
       <option id="shape_padding" type="int" default="0" />
       <option id="inner_padding" type="int" default="0" />
       <option id="trim" type="bool" default="false" />
+      <option id="grid" type="bool" default="false" />
```

Comment:
This option can be called `trim_by_grid` (in the future might be a possible `split_grid` so we have to avoid confusion)
```suggestion
      <option id="trim_by_grid" type="bool" default="false" />
```